### ISSUE TITLE
[macOS] Option to hide icons

### ISSUE THREAD

**Issue by AngryAnt** [keybase/client#5039]
Hey everyone, thanks for a great product!

I am really looking forward to replacing my various sync services with kbfs and to that end, I am working to replicate the same flow I have enjoyed with other services. One thing I really enjoy is keeping my overall UI to a bare minimum - just the information I actually need.

What I need from kbfs is my shortcut in the Finder favourites bar and a notification when something is incoming or was sent. The rest I can access by creating or deleting folders in Finder or via terminal.

Not everyone wants that flow, and that's fine. All I am asking for is an option to choose this flow for myself.

This is from Dash' UI preferences:
<img width="231" alt="screen shot 2016-12-08 at 09 21 10" src="https://cloud.githubusercontent.com/assets/102513/21002755/b1ae57da-bd27-11e6-8d08-61d0963b9da0.png">

I am pretty sure that the default values for both of these toggles is on. However the options were provided, allowing me to only see Dash UI when I need it - either via global shortcut or by re-launching the app (which just brings up the window of the already running instance).

IIRC the first time I saw this pattern was ten years ago in Quicksilver. Since then I have seen and used it in heaps of similar service apps.

I understand your reasoning behind https://github.com/keybase/client/issues/3606 and recognise the nerd-cred value in having a witty remark rather than actual settings in your settings panel. That said, at the end of the day surely you must realise the strength of a solution which allows for its users to, in an existing well-used pattern, choose their own flow rather than having it chosen for them as Google and Evernote Corp would?

Keybase GUI Version: 1.0.18-20161116190722+95f8303

**Comment by gabriel** [comment]
If you do File | Quit it will close the Keybase main window and dock icon will be hidden.
We don't have an option yet to not show the menu bar icon though.

cc @cecileboucheron @malgorithms

**Comment by AngryAnt** [comment]
Ah, interesting. Would I get the same effect from just removing Keybase from my login items or would that affect the functionality of Keybase? If not, we're halfway there! :)

**Comment by stephenyeargin** [comment]
Tagging some related issues I came across while looking through it.

- https://github.com/keybase/client/issues/3606
- https://github.com/keybase/client/issues/3907
- https://github.com/keybase/keybase-issues/issues/2391
- https://github.com/keybase/client/issues/2996

**Comment by alexreg** [comment]
Yes please! I asked for this issue before, and unfortunately it never got resolved. Would be a HIGHLY desirable feature for me.

**Comment by SirensOfTitan** [comment]
+1 here: the Keybase menu icon is completely useless for me, and takes up valuable menu bar real estate.
